skyportal/handlers/api/photometry.py

```diff
@@ -839,7 +839,7 @@ def put(self):
     def get(self, photometry_id):
         # The full docstring/API spec is below as an f-string
 
-        phot = Photometry.get_if_readable_by(photometry_id, self.current_user)
+        phot = Photometry.get_if_accessible_by(photometry_id, self.current_user)
```

Comment:
This and all other queries are updated in https://github.com/skyportal/skyportal/pull/1903, but since that is not yet merged, this was needed here to get the associated API test to pass

Reply:
Now that #1903 is merged, should this (and all other queries) be updated?

Reply:
This already is the updated access method